src/bosh-google-cpi/action/factory_interface.go

```diff
@@ -1,5 +1,5 @@
 package action
 
 type Factory interface {
-	Create(method string, ctx map[string]interface{}) (Action, error)
+	Create(method string, ctx map[string]interface{}, apiVersion int) (Action, error)
```

Comment:
Are API versions guaranteed to be ints?

Reply:
AFAIK, there's no plans to change it from ints.